libreyolo/backends/base.py

```
@@ -897,11 +1101,86 @@ def _parse_yolonas(self, all_outputs, orig_w, orig_h, conf, ratio=1.0):
+            keep = np.argpartition(-max_scores, YOLO_NAS_PRE_NMS_TOP_K - 1)[
+                :YOLO_NAS_PRE_NMS_TOP_K
+            ]
+            keep = keep[np.argsort(-max_scores[keep])]
+            boxes = boxes[keep]
+            max_scores = max_scores[keep]
+            class_ids = class_ids[keep]
+
+        input_h, input_w = _imgsz_hw(effective_imgsz)
+        resize_size = min(YOLO_NAS_RESIZE_SIZE, input_h, input_w)
+        ratio = min(resize_size / orig_h, resize_size / orig_w)
+        new_w = int(round(orig_w * ratio))
```

Ruff (0.15.17)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)